scripts/validation/validators.py

```diff
@@ -93,18 +90,9 @@ def eip712_schema_validator(data: str, filename: str) -> Tuple[bool, str]:
                 formatted,
             )
             with open(filename, "w") as f:
-                json.dump(loaded, f, indent=4, sort_keys=True, ensure_ascii=False)
-            return False, "Not formatted corrected"
+                f.write(formatted)
+            return False, "Not formatted, corrected"
```

Comment:
not formatted correctly?

Reply:
I understood it means "The document was not formatted, therefore it was corrected" :)

Reply:
I see :) thank you!